api/src/shared/helpers/objectHelper.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Flips an objects keys and values
+ */
+export function flip(obj: Record<any, any>): Record<any, any> {
+  const ret = {}
+  Object.keys(obj).forEach((key) => {
+    ret[obj[key]] = key
+  })
+  return ret
+}
+
+/** Deep merge two objects */
+export function deepMerge(obj1: object, obj2: object) {
+  const result = { ...obj1 } // Create a new object to avoid modifying the originals
+
+  for (const key in obj2) {
+    if (obj2.hasOwnProperty(key)) {
+      if (typeof obj2[key] === 'object' && typeof obj1[key] === 'object') {
+        // If both values are objects, recursively merge them
+        result[key] = deepMerge(obj1[key], obj2[key])
+      } else {
+        // Otherwise, overwrite the value from obj1 with the value from obj2
+        result[key] = obj2[key]
+      }
+    }
+  }
+
+  return result
+}
```

Comment:
Can we please move this to `/util/objects.util.ts` like I've done here for some `Map` util functionality? 🙏 

https://github.com/thegoodparty/gp-api/pull/9/files#diff-0e01814536c3a43f99f91a08ba84cc465f051550f1a248a89a60838c762be414R1-R8